themes/bootstrap3/templates/record/ajaxview-accordion.phtml

```diff
@@ -74,12 +74,12 @@
         if ($this->defaultTab === strtolower($tab)) { $tab_classes[] = 'default'; }
       ?>
       <div class="panel panel-default <?=implode(' ', $tab_classes) ?>">
-        <div id="<?=strtolower($tab)?>_<?=$idSuffix?>" class="list-tab-toggle panel-heading" data-toggle="collapse" data-parent="#accordion_<?=$idSuffix?>" data-target="#<?=strtolower($tab)?>_<?=$idSuffix?>-content"<?php if ($obj->supportsAjax() && in_array($tab, $this->backgroundTabs)):?> data-background<?php endif ?>>
+        <div id="<?=$this->escapeHtmlAttr(strtolower($tab))?>_<?=$idSuffix?>" class="list-tab-toggle panel-heading" data-toggle="collapse" data-parent="#accordion_<?=$idSuffix?>" data-target="#<?=strtolower($tab)?>_<?=$idSuffix?>-content"<?php if ($obj->supportsAjax() && in_array($tab, $this->backgroundTabs)):?> data-background<?php endif ?>>
```

Comment:
$idSuffix is already escaped. I'll change the name to indicate that.

Reply:
The one below is not the same, it has an additional `-content` suffix, so I'd keep them separate.

Reply:
Agreed, I just thought it might make sense to have an $idPrefix variable to make the difference between the two ids more obvious (since the lines are currently quite long). Fine to leave as-is, though!